### PR TITLE
Create a profile page and add Prettier 

### PR DESCRIPTION
**Pull request by ronal2do**

### CODE REVIEW

**Comment by alexandru-paduraru** [comment]
@ronal2do thank you for the PR. We will check it and if everything is working and looks great we will add it to the master branch. We will come soon with feedback.

Best,
Alex

File: src/containers/Profile/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+              backgroundSize: 'cover',
+              backgroundPosition: 'top center'
+            }}
+          />
+
+          <div className="main main-raised">
+            <div className="profile-content">
+              <Grid>
+                <Row>
+                  <div className="profile">
```
Comment (aluciu) [inline src/containers/Profile/Profile.jsx]: maybe we can make a widget for this piece of code
_Eg._
```
<Profile
  img={...}
  name="Christian Louboutin"
  role="Designer"
/>
```

File: src/containers/Profile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                              <i className="material-icons">favorite</i>
+                              Favorite
+                            </NavItem>
+                          </Nav>
+
+                          <Tab.Content animation={false} className="gallery">
+                            <Tab.Pane eventKey="studio">
+                              <Row>
+                                <Col md={6}>
+                                  <img
```
Comment (aluciu) [inline src/containers/Profile/Profile.jsx]: for the image elements we can use the `<Image />` component from [react-bootstrap](https://react-bootstrap.github.io/components.html#images)

```
<Image 
  src={chris1} 
  rounded
  responsive
/>
```

File: src/containers/Profile/Profile.jsx (hunk 1 above)
Comment (ronal2do) [inline src/containers/Profile/Profile.jsx]: Great!

You will need to refactor and create more components of this type, I will try to mount as much as I can

**Pull request merged**